docs/product-manuals/cloud-console/manage-organization/manage-users.md

```diff
@@ -29,6 +29,8 @@ The following table illustrates the rights of each role:
 | Billing: Request paid plan   | X     | X     |        |
 | Users: Manage admins         | X     |       |        |
 
-Users are invited to a Camunda Cloud organization via their e-mail address, which must be accepted by the user. As long as the invitation has not been accepted, the user remains in the 'Pending' state.
+Users are invited to a Camunda Cloud organization via their email address, which must be accepted by the user. As long as the invitation has not been accepted, the user remains in the `Pending` state.
 
-People can also be invited to an organization that does not yet have a Camunda Cloud account. In this case the invited person must first create a Camunda Cloud account and can then gain access to the organization.
+People can also be invited to an organization that does not yet have a Camunda Cloud account. In this case, the invited individual must first create a Camunda Cloud account, and can then gain access to the organization.
+
+[//]:# (Are we trying to say above that people can be invited to an organization with a Camunda Cloud account even if the invited person doesn't have an account? If this is the case, I'd say "People who do not yet have a Camunda Cloud account can also be invited to an organization.")
```

Comment:
Exactly, your proposal is much better: `People who do not yet have a Camunda Cloud account can also be invited to an organization.`

Reply:
Awesome -- thanks for clarifying.